docs/UserGuide.md

```diff
@@ -238,37 +238,38 @@ Format: `list`
 
 ### 7.4 Editing a student : `edit`
 
-Edits an existing person in the list of students.
+Edits an existing person in the list of students. Exsisting fields are updated to match the specified input fields.
 
 Format: `edit STUDENT_ID [id/STUDENT_ID] [n/NAME] [e/EMAIL] [g/GRADE] [gp/GROUP_NAME]...`
 
-* Edits the person with the specified `STUDENT_ID`.
```

Comment:
Do you think it would be good to keep this line as it explains the purpose of specifying the student id after the command word

Reply:
Yes, but I will move it upwards to conform to the format

1. What the command does
2. The format of the command
3. Valid values that the command can take in
4. Example usages
5. _Optional Screenshot_

So it will be in 1.